bee-hive/bee_hive/workflow.py

```diff
@@ -60,15 +60,21 @@ def run(self):
             print("not supported yet")
 
     def _sequence(self):
-        prompt = self.workflow["spec"]["template"]["prompt"]
-        for agent in self.agents.values():
-            if (
-                self.workflow["spec"]["strategy"]["output"]
-                and self.workflow["spec"]["strategy"]["output"] == "verbose"
-            ):
-                prompt = agent.run_streaming(prompt)
-            else:
-                prompt = agent.run(prompt)
+        prompt = self.workflow["spec"].get("prompt","")   # using get b/c prompt is optional
+        steps = self.workflow["spec"].get("steps",[])
```

Comment:
Steps are in `spec.template`, so I think you may need to do

```suggestion
        steps = self.workflow["spec"]["template"].get("steps",[])
```

(syntax may not be exactly correct)

Reply:
How come there should be template? I'm basing it off the latest workflow_schema merged, and if we look at that the organization is like this:
specs
    template
    agents
    prompt
    steps

so from what I see, it doesnt seem necessary

Reply:
you're right, we updated the schema but not the test